kafka-rest/src/main/java/io/confluent/kafkarest/ratelimit/StacklessCompletionException.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest.ratelimit;
```

Comment:
I would put this in `kafkarest.common` instead, where `CompletableFutures` is.

Reply:
Good suggestion, but now that you bring it, I think `kafkarest.exceptions` is an even better place for this.